jobs/build/el8-rebuilds/Jenkinsfile

```diff
@@ -51,32 +51,35 @@ node {
 
     def version = params.BUILD_VERSION
     currentBuild.displayName += " - ${version}"
-    try {
-        buildlib.kinit()
-        def builds = [
-            "openshift": {
-                stage("openshift RPM") {
-                    commonlib.shell("./rebuild_rpm.sh openshift ${version}")
+    def branch = buildlib.doozer("--group openshift-${version} config:read-group branch", [capture: true]).trim()
+    if (branch.endsWith('-rhel-7')) {
```

Comment:
what isn't obvious here is that we are in fact still building the two RPMs for rhel7 first and then rebuilding for rhel8. See https://github.com/openshift/ocp-build-data/blob/128e7f81fe8b37af5c0a0b5816597471b9a29adf/rpms/openshift.yml#L18-L20 and https://github.com/openshift/ocp-build-data/blob/128e7f81fe8b37af5c0a0b5816597471b9a29adf/rpms/openshift-clients.yml#L12-L14

We need the RPMs for both since we support RHEL 7 workers. It would seem a little less crazy to build on rhel8 first for 4.6+, or to try to build both at the same time, but the simplest _logic_... is to keep doing what we've been doing.